packages/image-block/src/components/helpers.ts

```diff
@@ -90,7 +90,7 @@ export const getTotalImagePadding = (blockSettings: Settings): CSSProperties =>
 const getPadding = (blockSettings: Settings): string =>
     blockSettings.hasCustomPadding ? blockSettings.paddingCustom : paddingValues[blockSettings.paddingChoice];
 
-export const getImageStyle = (blockSettings: Settings, width: string): CSSProperties => {
+export const getImageWrapperStyle = (blockSettings: Settings): CSSProperties => {
```

Comment:
https://www.loom.com/share/7c292bd3939a4d278eeb1147479cdc46 The background appears to be visible on viewmode underneath long images

Reply:
Thanks, fixed by making the container flex.